### PR TITLE
Fix package.json

### PR DESCRIPTION
**Pull request by Aichnerc**
The name, GitHub URL and version has been fixed.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the package name, GitHub repository name and version are not correct.

**What is the new behavior (if this is a feature change)?**
- The package name, GitHub repository name and version have been corrected.

### CODE REVIEW

**Review by LucaAllmaier — APPROVED** [review]
Incredible work.

**Review by Aichnerc — COMMENTED** [review]
LGTM 🥫

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by kleberbaum — APPROVED** [review]
Should be very ok to merge.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-mdb-template",
-  "version": "1.2.3",
+  "name": "cloud-homepage",
+  "version": "1.0.0",
```
Comment (kleberbaum) [inline package.json:3]: But y are going back to 1.0.0?

**Review by kleberbaum — APPROVED** [review]
Should still be ok to merge.

**Pull request merged**